scripts/setup-project.sh

```diff
@@ -18,135 +18,108 @@ case "$name" in
   same54xpro)
     pushd "$(dirname $0)"/../samples/"${name}"
 	;;
-  rt1060-nxp)
-    pushd "$(dirname $0)"/../samples
-	mkdir -p rt1060
-	popd >/dev/null
-	pushd "$(dirname $0)"/../samples/rt1060
-	;;
-  maaxboardrt-nxp)
-    pushd "$(dirname $0)"/../samples
-	mkdir -p maaxboardrt-nxp
-	popd >/dev/null
-	pushd "$(dirname $0)"/../samples/maaxboardrt-nxp
-	;;
-  lpc55s69-nxp)
-    pushd "$(dirname $0)"/../samples
-	mkdir -p lpc55s69
-	popd >/dev/null
-	pushd "$(dirname $0)"/../samples/lpc55s69
-	;;
   maaxboardrt)
     pushd "$(dirname $0)"/../samples/"${name}"
 	;;
 esac
 
 # initial cleanup
 
-rm -rf iotc-c-lib cJSON b-l4s5i-iot01a mimxrt1060 same54Xpro stm32l4 maaxboardrt
+rm -rf iotc-c-lib cJSON libTO b-l4s5i-iot01a mimxrt1060 same54Xpro stm32l4 maaxboardrt
```

Comment:
Think I saw a comment about removing dependency on TO ?

Reply:
Removing dependency to lib_TO in **that** file. Not all together.